lib/get_parser.py

```diff
@@ -427,6 +448,12 @@ def get_parser() -> argparse.ArgumentParser:
         ' +DESIRED_CVMFS="OSG" to the job classad attributes and'
         " '&&(CVMFS==\"OSG\")' to the job requirements",
     )
+    parser.add_argument(
```

Comment:
Just a thought: should we put this in the base parser, so we can set the schedd for jobsub_q, etc.?

Reply:
Sure - I can move it there, and then we can implement it in a separate issue?

Reply:
Sorry - that was unclear.  I can move the argument to the base parser, and then we can implement the functionality for jobsub_q in a separate issue?

Reply:
So this actually already exists, by passing the `-name` flag.  There's just a bug in the code right now that ignores the `-name` flag:

```
$ jobsub_q -G fermilab  -name jobsub01.fnal.gov sbhat
JOBSUBJOBID                             OWNER           SUBMITTED     RUNTIME   ST PRIO   SIZE  COMMAND
688176.0@jobsub04.fnal.gov              sbhat           05/16 16:15   0+00:19:53 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
688179.0@jobsub04.fnal.gov              sbhat           05/16 16:15   0+00:00:22 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
503632.0@jobsub05.fnal.gov              sbhat           05/11 16:10   0+00:01:07 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
677046.0@jobsub05.fnal.gov              sbhat           05/19 19:50   0+00:18:43 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
61563115.0@jobsub01.fnal.gov            sbhat           05/10 16:46   0+00:00:08 C    0    0.0 simple.sh 
61859314.0@jobsub01.fnal.gov            sbhat           05/19 19:50   0+00:00:22 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
68343522.0@jobsub02.fnal.gov            sbhat           05/11 16:10   0+00:17:53 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
```

It's a quick fix, and with the next commit, it works right, with the `-name` flag and not the `--schedd-for-testing` flag:

```
$ jobsub_q -G fermilab  -name jobsub01.fnal.gov sbhat
JOBSUBJOBID                             OWNER           SUBMITTED     RUNTIME   ST PRIO   SIZE  COMMAND
61563115.0@jobsub01.fnal.gov            sbhat           05/10 16:46   0+00:00:08 C    0    0.0 simple.sh 
61859314.0@jobsub01.fnal.gov            sbhat           05/19 19:50   0+00:00:22 C    0    0.0 dagman_wrapper.sh -p 0 -f -l . -Lockfi
```

Reply:
Actually - I'll make that a separate PR.  I'll move the `--schedd-for-testing` flag to the base parser, and if you approve, we'll merge it.  I'll open another PR for the bugfix for the issue immediately above, just so we can track the progress more easily and it doesn't all become one huge merge commit eventually.